app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -56,6 +57,8 @@ class SelectionInteractionViewModel private constructor(
     computeChoiceItems(choiceSubtitledHtmls, hasConversationView, this)
 
   private val isAnswerAvailable = ObservableField(false)
+  val selectedItemText = ObservableField("Please select all correct choices")
+  val selectedItemsList: List<ObservableBoolean>? = null
```

Comment:
This also needs to be initialized to be a list of observable booleans for each possible choice, e.g. something like:

```kotlin
val selectedItemsList by lazy { List(choiceItems.size) { ObservableBoolean() } }
```